packages/invoices/src/OutboundShipment/DetailView/columns.ts

```diff
@@ -82,7 +82,6 @@ export const useOutboundColumns = ({
       [
         'itemName',
         {
-          maxWidth: 400,
```

Comment:
as far as I could see, all columns were fixed, and we want the table to stretch

Reply:
Same comment as above. We do want the table to stretch, but if a particular item has a name that fills most of the screen, we probably want to put a limit on it at some point. I think `maxWidth` is fine, as the column will still stretch appropriately, it just won't let it get ridiculous! 😄

Maybe `maxWidth: 650`?